### ISSUE TITLE
Failed $BASH validation in `slack/status` when bash is manually installed via `apk add`

### ISSUE THREAD

**Issue by ekanai** [CircleCI-Public/slack-orb#66]
### Orb version
3.2.0

### What happened
I build `slack/status` on alpine base image. But failed $BASH validation when bash was installed from `apk add`.
So, slack post has success after validation was failed.

<img width="1573" alt="スクリーンショット 2019-07-16 18 37 09" src="https://user-images.githubusercontent.com/5866443/61283878-c926c500-a7f8-11e9-8ec0-7c56df6724c5.png">

I want use this orb on alpine 😄 

For example, how is this method?

```sh
if [ -z "$(type bash)" ]; then
  echo Bash not installed.
  exit 1
fi
```

**Comment by iynere** [comment]
@ekanai try `apk add bash`

**Comment by iynere** [comment]
@ekanai ahh, sorry, i should read before replying 😅

**Comment by iynere** [comment]
@ekanai we'll work to prioritize making this orb compatible with alpine, thank you !

**Comment by samsalisbury** [comment]
I'm seeing this too. @iynere I think requiring bash isn't too bad (though would be nice to just rely on POSIX shell and avoid the need eventually), it's just the test for whether bash is installed isn't right for Alpine. I'll send a PR to address this, as right now I'm having to explicitly set `BASH: 1` in my environment declaration as a workaround, which is just noise in my config.

**Comment by samsalisbury** [comment]
@iynere this PR fixes the issue: #72

**Comment by zehrakomurcu** [comment]
I had the same problem. Setting bash env variable in the job you're using alpine is worked for me.
```
environment:
      BASH: true
```
